src/components/RangeSlider.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Slider } from "antd";
+
+const min = 1818,
```

Comment:
please consider to fetch min year in range and max year in range from the api together with `sunspot data` or by another request